scherztc/decidim_application
# Aienable-43: Link nvmrc to node version to keep nvmrc up to date with decidims requirements

Jira Issue: https://ucdts.atlassian.net/browse/AIENABLE-43

This PR adds an .nvmrc file in as a symlink to the .node-version file that is provided by Decidim.  It makes available the command `nvm use`, which relies on an .nvmrc file being present.

To test this, 

1. Open up the codebase and look at the .nvmrc file.  It should be the same as the .node-version file. 
2. Modify the .node-version file to a different version of node.  The .nvmrc file should automatically update to the change. 
3. Modify the .nvmrc file back to the original version.  The .node-version file should be changed back to its original form.

Note:  this change is only supported for Unix/Linux/macOS users.  Windows does not support symlinks at this time.

Since our group all uses MacOS and we are setting up a server using Unix/Linux, this is a simple solution to keeping the .nvmrc and .node-version files in sync.  If the .nvmrc file is overwritten by a Windows user, it will become a plain text file and the sym link will be broken.
